src/components/recommended-section/recommended-content-section.tsx

```diff
@@ -0,0 +1,55 @@
+import { Grid } from 'theme-ui';
+import Card, { getCardProps } from '../card';
+import { ContentItem } from '../../interfaces/content-item';
+import AdditionalContentPlaceholder from './additional-content-placeholder';
+
+interface RecommendedSectionProps {
+    content?: ContentItem[];
+}
+
+const placeholderStyles = (contentLength: number) => {
+    const gridColumnValues = [
+        ['auto', null, null, '2 / span 2', null, '2 / span 3'],
+        ['auto', null, '1 / span 2', 'auto', null, '3 / span 2'],
+        'auto',
+    ];
+
+    return { gridColumn: gridColumnValues[contentLength - 1] };
+};
+
+const RecommendedContentSection: React.FunctionComponent<
+    RecommendedSectionProps
+> = ({ content = [] }) => {
+    const totalItems = Math.min(content.length + 1, 4);
+
+    return (
+        <Grid
+            // If full 4 items (incl placeholder), never show a 3 column layout
+            columns={[1, null, 2, totalItems === 4 ? 2 : 3, null, 4]}
+            gap="inc30"
+            sx={{
+                overflow: 'visible',
+            }}
+        >
+            {content.map((item, i) => (
+                <div key={i} data-testid="recommended-content-item">
```

Comment:
it looks like `Card` sets `data-testid={`card-${variant}`}` on the parent element in the component, could we possibly try to query for that in the tests instead so we can wrapping this is another `<div />` ?

Reply:
good catch, didn't notice that attribute was already there